src/sgame/sg_main.cpp

```diff
@@ -2449,6 +2452,18 @@ void CheckExitRules()
 		LogExit( "Aliens win." );
 		G_MapLog_Result( 'a' );
 	}
+	else if ( g_emptyTeamsSkipMapTime.integer &&
+		( level.time - level.startTime ) / 60000 >=
+		g_emptyTeamsSkipMapTime.integer &&
+		level.team[ TEAM_ALIENS ].numClients == 0 && level.team[ TEAM_HUMANS ].numClients == 0 )
```

Comment:
There are `numClients` (everyone), `numPlayers` (only real players) and `numBots` (only bots). You would want to check for `numPlayers`.